html/controls/operator.js

```diff
@@ -1166,8 +1170,6 @@ function createTimeoutDialog() {
 			firstJamListed[i] = 0;
 			lastJamListed[i] = 0;
 			jamDropdownTemplate[i] = $('<select>').attr('id', 'JamDropdown').attr('period', i);
-			WS.Register(['ScoreBoard.Period('+i+').CurrentJamNumber', 'ScoreBoard.Period('+i+').FirstJamNumber'], processJamNumber);
-			WS.Register(['ScoreBoard.Period('+i+').Timeout'], processTimeout);
 			lastPeriodRegistered = i;
```

Comment:
This creates a race. If for a completed period both calls to `processJamNumber` are made before this code is run, the period will not be initialised properly.

Reply:
I've reworked this.